app/models/accounting/quickbooks/data_extractor.rb

```diff
@@ -5,8 +5,14 @@ def initialize(object)
         @object = object
 
         case @object.qb_object_type
-        when "JournalEntry", "Purchase", "Deposit", "Bill"
+        when "JournalEntry"
```

Comment:
Should I use constantize on "Accounting::Quickbooks::#{@object.qb_object_type}Extractor" instead of switch statement?

Reply:
I like the explicitness of the switch, and it should keep us from accidentally calling an extractor that doesn't exist, otherwise we'd need to write some other code to check that `qb_object_type` has a matching extractor class before we call it.